packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -40,7 +40,7 @@ const getImageSize = file => {
 // There is no guarantee that the module resolved is the module executing!
 // This can occur in mono repos depending on how dependencies have been hoisted.
 // The direct require has been left only to avoid breaking changes.
-let { boundActionCreators } = require(`gatsby/dist/redux/actions`)
+let boundActionCreators
```

Comment:
This implementation is weird. I know it’s not the scope of this. But can you explain to me why we inject the boundActionCreators? Is this the whole local Gatsby vs global Gatsby stuff?

Reply:
`boundActionCreators` was deprecated in v2. We can continue using it, but it’s recommended that one rename it to `actions`. Note that `actions` is what is passed _in_ in most lifecycle APIs. 

So the name `boundActionCreators` is largely a remnant of the past.